### PR TITLE
Flutter SDK docs page updates

### PR DESCRIPTION
**Pull request by shahbaz17**

### CODE REVIEW

File: docs/sdk/flutter/initialize.mdx [hunk 1]
```diff
@@ -5,50 +5,119 @@ displayed_sidebar: sdk
 description: "Flutter SDK - Initialization | Documentation - Web3Auth"
 ---
 
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
 ---
```
Comment (yashovardhan) [inline docs/sdk/flutter/initialize.mdx:11]: The flutter guide has more initialisation and installation steps. Are we not adding them here?

**Review by yashovardhan — CHANGES_REQUESTED** [review]
I feel generally this SDK reference looks good, all the things are similar to other mobile sdk reference except the code snippets, so I don't think much review is required there.

The only question I have is regarding initialisation and installation. The guide features more steps as I see. Have you added all the steps here as well?

File: docs/sdk/flutter/initialize.mdx (hunk 1 above)
Comment (shahbaz17) [reply docs/sdk/flutter/initialize.mdx:11]: Added.

**Comment by shahbaz17** [comment]
Covers both Android and iOS builds.

**Review by yashovardhan — APPROVED** [review]
LGTM

**Pull request merged**